beacon-chain/rpc/prysm/v1alpha1/validator/duties_test.go

```diff
@@ -32,7 +32,7 @@ func pubKey(i uint64) []byte {
 	return pubKey
 }
 
-func TestGetDuties_OK(t *testing.T) {
+func TestGetDutiesV2_OK(t *testing.T) {
```

Comment:
This seems to replace the GetDuties original tests

Reply:
adding them back in with new duties tests